dataflow/src/main/java/org/checkerframework/dataflow/cfg/StringCFGVisualizer.java

```diff
@@ -25,7 +25,7 @@
     @Override
     public Map<String, Object> visualize(
             ControlFlowGraph cfg, Block entry, @Nullable Analysis<V, S, T> analysis) {
-        String stringGraph = visualizeGraph(cfg, entry, analysis);
+        String stringGraph = visualizeGraph(cfg, entry, analysis).trim();
```

Comment:
This will be improved by https://github.com/typetools/checker-framework/pull/3395/files#diff-b5fa6abf87cb7ac00c95e235720bfa66R50-R56. `.trim()` seems not the best solution here.
The reason I added `trim()` is to remove the extra blank line at the end of the string graph. This blank line is generated by visualizing special blocks (exit or special-exit). If we left this extra blank line, `Excepted.txt` will fail the basic style test (too many blank lines at the end of the file).